.github/workflows/ci.yml

```diff
@@ -214,6 +214,11 @@ jobs:
           os: ubuntu-latest
           env: {CXX: clang++, CC: clang,  ALPAKA_CI_CLANG_VER: 10,     ALPAKA_CI_STDLIB: libc++,    CMAKE_BUILD_TYPE: Release, ALPAKA_CI_BOOST_BRANCH: boost-1.73.0, ALPAKA_CI_CMAKE_VER: 3.15.7, OMP_NUM_THREADS: 4, ALPAKA_CI_DOCKER_BASE_IMAGE_NAME: "ubuntu:20.04", ALPAKA_ACC_CPU_BT_OMP4_ENABLE: OFF}
 
+        # icpc
+        - name: linux_icpc_release
+          os: ubuntu-latest
+          env: {TRAVIS_OS_NAME: linux,   CXX: icpc,    CC: icc, CMAKE_BUILD_TYPE: Release, ALPAKA_CI_BOOST_BRANCH: boost-1.73.0, ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE: OFF, ALPAKA_CI_CMAKE_VER: 3.15.7, OMP_NUM_THREADS: 2, ALPAKA_CI_STDLIB: libstdc++, ALPAKA_CI_DOCKER_BASE_IMAGE_NAME: "ubuntu:18.04"}
```

Comment:
May we have to disable the fibers backend because we would be mixing gcc built boost libs with an intel compiler built executable?